lib/plugins/aws/deploy/lib/mergeIamTemplates.js

```diff
@@ -36,16 +36,76 @@ module.exports = {
         .Properties
         .PolicyName = `${this.options.stage}-${this.serverless.service.service}-lambda`;
 
-      iamPolicyLambdaExecutionTemplate
-        .IamPolicyLambdaExecution
-        .Properties
-        .PolicyDocument
-        .Statement[0]
-        .Resource = `arn:aws:logs:${this.options.region}:*:*`;
-
       _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Resources,
         iamPolicyLambdaExecutionTemplate);
 
+      if (!this.serverless.service.provider.cf_logs) {
+        this.serverless.service.provider.compiledCloudFormationTemplate.Resources
+          .IamPolicyLambdaExecution
+          .Properties
+          .PolicyDocument
+          .Statement[0]
+          .Resource = `arn:aws:logs:${this.options.region}:*:*`;
+
+        this.serverless.service.provider.compiledCloudFormationTemplate.Resources
+          .IamPolicyLambdaExecution
+          .Properties
+          .PolicyDocument
+          .Statement[1]
+          .Resource = `arn:aws:logs:${this.options.region}:*:*`;
+
+        const warningMessage = [
```

Comment:
This `warningMessage` should be updated as it looks like this (formatting, comma and fullstops):

![bildschirmfoto 2016-10-24 um 13 35 16](https://cloud.githubusercontent.com/assets/1606004/19644743/bf1ba07a-99f1-11e6-9e02-ab008f6ae042.png)

Furthermore I would change `CF` to `CloudFormation` to reduce cognitive load.